lib/modules/dosomething/dosomething_taxonomy/dosomething_taxonomy.module

```diff
@@ -199,12 +199,15 @@ function dosomething_taxonomy_get_partners_data($field_partners, $logo_style = '
  *
  * @param array $field_partners
  *   An loaded field_partners field array.
+ * @param string $language_code
+ *   The optional language code which indicates to only return partners
+ *   translated in that lang.
  *
  * @return mixed
  *   Multi-dimensional array of partners/sponsors, or NULL if empty.
  *
  */
-function dosomething_taxonomy_get_partners_vars($field_partners = array()) {
+function dosomething_taxonomy_get_partners_vars($field_partners = array(), $language_code = NULL) {
```

Comment:
true, apart of me was wondering if we'd have a scenario where we didn't want to filter by language. but if you think its safe to remove the default then 👍